resdk/tests/unit/test_resolwe.py

```diff
@@ -122,14 +122,22 @@ def test_env_variables(self, resauth_mock):
         self.assertEqual(resauth_mock.call_args[0][1], None)
 
         # If environment variable is set, it overrides the default URL.
-        os.environ['RESOLWE_API_HOST'] = 'http://resolwe-api:8000'
+        os.environ['RESOLWE_HOST_URL'] = 'http://resolwe-api:8000'
         os.environ['RESOLWE_API_USERNAME'] = 'foo'
         os.environ['RESOLWE_API_PASSWORD'] = 'bar'
         resolwe_api = Resolwe()
         self.assertEqual(resolwe_api.url, 'http://resolwe-api:8000')
         self.assertEqual(resauth_mock.call_args[0][0], 'foo')
         self.assertEqual(resauth_mock.call_args[0][1], 'bar')
 
+        os.environ['RESOLWE_API_HOST'] = 'http://resolwe-api:9000'
+        os.environ['RESOLWE_API_USERNAME'] = 'foo'
+        os.environ['RESOLWE_API_PASSWORD'] = 'bar'
+        resolwe_api = Resolwe()
+        self.assertEqual(resolwe_api.url, 'http://resolwe-api:9000')
```

Comment:
Since `RESOLWE_HOST_URL` should take precedence, the URL should equal `'http://resolwe-api:8000'`.

Add another test like:
```
os.environ.pop('RESOLWE_HOST_URL', None)
resolwe_api = Resolwe()
self.assertEqual(resolwe_api.url, 'http://resolwe-api:9000')
```